apps/web/components/sign-up/confirm-signup-modal.tsx

```
@@ -0,0 +1,59 @@
+      >
+        <DialogTitle className="sr-only">
+          {confirmSignupModal('title')}
+        </DialogTitle>
+
+        <ModalHeader text={confirmSignupModal('title')} />
+
+        <section className="bg-white px-[4rem] py-[7rem]">
+          <div className="text-center">
+            <p
+              className="inter-body3 font-medium text-gray-800"
+              dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)